### DISCUSSION TITLE
Cual es la diferencia entre git y github?

### DISCUSSION THREAD

**Discussion by Pedro6094** [hackatomik/SISCOx256#43] [Git y GitHub | Lab]
Me encantaría saber que aspectos difieren el término git con el de github

**Comment by quantumquirkxyz** [comment]
Git es una herramienta que usas en tu computadora para llevar un control sobre los cambios en el código de tus proyectos. Imagina que estás trabajando en un proyecto y quieres guardar cada versión del código para poder regresar a una versión anterior si algo sale mal. Eso es lo que hace Git: guarda un historial de los cambios que haces en tu código.

GitHub, por otro lado, es como una especie de red social para programadores. Es un sitio web donde puedes subir tu código para que otros puedan verlo y colaborar contigo. En GitHub, puedes almacenar tu código en la nube, compartirlo con tu equipo, recibir sugerencias, y ver el trabajo de otros.

**Reply by quantumquirkxyz** [reply]
Aquí puedes tener una descripción más detallada: 
[Lecture Note {Git y GitHub}.pdf](https://github.com/user-attachments/files/16914235/Lecture.Note.Git.y.GitHub.pdf)
